README.md

```diff
@@ -189,6 +189,14 @@ The [default configurations](src/main/resources/application.conf) can be overwri
       - `SOURCE_S3_OBJECTKEY` The Object containing the ACL CSV in S3
     - `io.conduktor.ksm.source.BitbucketServerSourceAcl`: get the ACL from Bitbucket Server using the v1 REST API. Great if you have private repos in Bitbucket.
     - `io.conduktor.ksm.source.BitbucketCloudSourceAcl`: get the ACL from Bitbucket Cloud using the Bitbucket Cloud REST API v2.
+    - `io.conduktor.ksm.source.HttpSourceAcl`: get the ACL from an HTTP endpoint. You can enable [Google OAuth OIDC Token Authentication](https://cloud.google.com/docs/authentication/production).
+        - `SOURCE_HTTP_URL` HTTP endpoint to retrieve ACL data.
+        - `SOURCE_HTTP_METHOD` HTTP Method. Default is `GET`.
+        - `SOURCE_HTTP_ENABLE_AUTH` If enable Google OICD Authentication. Default is `false`.
```

Comment:
can we make the auth more generic in case more auth mechanisms are added in the future? (bearer etc...)

Reply:
I thought about that, in fact, if we have a `pluggable` authentication method, I think you could even consolidate others HTTP sources like Github/Gitlab as they all are HTTP too. 
What you propose? An authentication class implementation for HTTP Source for example? How could we handle specifics configuration for this authentication class?